applications/api.py

```diff
@@ -98,7 +98,10 @@ def process_upload_resume(resume_file, bootcamp_application):
     """
     if bootcamp_application.state == AppStates.AWAITING_PROFILE_COMPLETION.value:
         raise InvalidApplicationException("The BootcampApplication is still awaiting profile completion")
-    bootcamp_application.upload_resume(resume_file)
+    if resume_file:
+        bootcamp_application.upload_resume(resume_file)
```

Comment:
A user can provide either or both of the resume file and linkedin URL, and we only want to change the application state once if they provide both, so I think it makes the most sense to combine this into a single method that accepts both items as kwargs. Signature could be something like `add_work_history(self, *, resume_file=None, linkedin_url=None)`